pkg/acsengine/defaults-controller-manager.go

```diff
@@ -44,10 +44,11 @@ func setControllerManagerConfig(cs *api.ContainerService) {
 
 	// Default controller-manager config
 	defaultControllerManagerConfig := map[string]string{
-		"--node-monitor-grace-period":   DefaultKubernetesCtrlMgrNodeMonitorGracePeriod,
-		"--pod-eviction-timeout":        DefaultKubernetesCtrlMgrPodEvictionTimeout,
-		"--route-reconciliation-period": DefaultKubernetesCtrlMgrRouteReconciliationPeriod,
-		"--terminated-pod-gc-threshold": DefaultKubernetesCtrlMgrTerminatedPodGcThreshold,
+		"--node-monitor-grace-period":       DefaultKubernetesCtrlMgrNodeMonitorGracePeriod,
+		"--pod-eviction-timeout":            DefaultKubernetesCtrlMgrPodEvictionTimeout,
+		"--route-reconciliation-period":     DefaultKubernetesCtrlMgrRouteReconciliationPeriod,
+		"--terminated-pod-gc-threshold":     DefaultKubernetesCtrlMgrTerminatedPodGcThreshold,
+		"--use-service-account-credentials": DefaultKubernetesCtrlMgrUseSvcAccountCreds,
```

Comment:
I think this should be removed since it is controlled by RBAC at the botton of this file.

```
 	if *o.KubernetesConfig.EnableRbac {
 		o.KubernetesConfig.ControllerManagerConfig["--use-service-account-credentials"] = "true"		 		o.KubernetesConfig.ControllerManagerConfig["--use-service-account-credentials"] = "true"
 	}
```

Reply:
My concern is that it changes the default behavior of "--use-service-account-credentials", it used to be false when RBAC is off, but now it becomes true.

Reply:
@pidah Can you explain the use-case where a cluster would want `--use-service-account-credentials` without RBAC enabled? In other words, what is the justification for disentangling this controller-manager runtime option from higher order acs-engine RBAC enforcement?

Reply:
@jackfrancis @JunSun17 No strong opinion – as I understand it, the `--use-service-account-credentials` flag ensures the kubernetes controller manager uses the individual designated service accounts for each control loop instead of a single service account credential for all control loops. This segregation/subdivision seemed like a good default and shouldn't impact the cluster when RBAC is not enabled.